crates/miden-objects/src/transaction/tx_args.rs

```diff
@@ -54,6 +55,7 @@ impl TransactionArgs {
             note_args: Default::default(),
             advice_inputs: AdviceInputs::default().with_map(advice_map),
             foreign_account_inputs,
+            auth_arg: EMPTY_WORD,
```

Comment:
Nit: Should we prefer using `Word::empty()` instead of `EMPTY_WORD`? Recently we were moving in a direction of using associated constants rather than free ones, but not super important.

Reply:
We widely use the `EMPTY_WORD` in assembly code, so in that case probably we could leave it just for the consistency. But that's not a strong preference.